### ISSUE TITLE
Tables

### ISSUE THREAD

**Issue by dubefab** [primary-theme/obsidian#205]
Is it normal if my tables look like this?
I don't have snippets modifying anything.
<img width="402" alt="image" src="https://github.com/primary-theme/obsidian/assets/36094769/fa24be43-2868-4974-9b90-f0c5e47f1161">

**Comment by ceciliamay** [comment]
yes it is normal! I'm planning to add back the borders on the sides and add a colored background for the table headers.

**Comment by ceciliamay** [comment]
the tables now look like this in the latest update.

![Image](https://github.com/primary-theme/obsidian/assets/90643719/f879f08d-8b0e-42f2-b289-7dadde5d472b)

in the future i will add the option to have alternating row colors for better readability.
